scripts/release_scripts/update_configs.py

```diff
@@ -110,11 +110,11 @@ def check_updates_to_terms_of_service(
     g = github.Github(personal_access_token)
     repo = g.get_organization('oppia').get_repo('oppia')
 
-    common.open_new_tab_in_browser_if_possible(TERMS_PAGE_URL)
+    common.open_new_tab_in_browser_if_possible(TERMS_PAGE_FOLDER_URL)
     python_utils.PRINT(
         'Are the terms of service changed? Check commits/changes made '
-        'to the file: terms-page.mainpage.html. Enter y/ye/yes if they '
-        'are changed else enter n/no.')
+        'to the files in: core/templates/pages/terms-page. '
```

Comment:
We are expecting to only check the changes in the HTML (or text in the html) file, right? Any specific reason for this change?

Reply:
Firstly we should probably check both terms-page.mainpage.html and terms-page.component.html but I also think that it is useful to check all the changes for safety reasons (change to TS file can modify the content of HTML). But let me know what do you think.

Reply:
The mainpage.html doesn't have any user-facing text, right? But I agree if we won't track the directory then we will have to track and update this place for any structural changes happening in the future. (I'm fine with either.)

Reply:
Okay, keeping it for the whole folder.